.github/workflows/ci-macarm.yml

```diff
@@ -0,0 +1,153 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+#
+# GitHub Actions workflow file
+# https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: CI-MacARM
+
+# This workflow is for CI for ARM-based MacOS. In an ideal world, this would
+# just be another matrix entry in the main CI workflow, but MacOS ARM runners
+# too expensive to run on every PR push. So we break into a separate workflow
+# to have more fine-grained control over when it runs.
+
+# This section controls when the workflow runs.
+on:
+  # Run upon push, but only for tagged versions, or if the branch name
+  # contains the substrings "macarm", "-alpha", "-beta", or is called
+  # "release".
+  push:
+    tags:
+      - v**
+    branches:
+      - '*macarm*'
+      - '*alpha*'
+      - '*beta*'
+      - 'release'
+  # Run for pull requests whose branch name includes "macarm" (this allows
+  # us to test specific PRs that we think need ARM verification).
+  pull_request_target:
```

Comment:
I'm unclear on why this PR didn't generate a workflow and run.  Is it because you're using pull_request_target rather than pull_request here?

You can see Larry's PR that first added the Yaml workflow [ran the macos-13-xlarge job](https://github.com/AcademySoftwareFoundation/OpenImageIO/actions/runs/6528102304).

Reply:
He mentioned that he was able to get the workflow running as admin.

Reply:
Switching to pull_request also skipped the execution of the new workflow. We will need to approve the PR to the main branch so the workflow is available for execution from my fork.